src/VisualStudio/Core/Def/Implementation/Interop/WrapperPolicy.cs

```diff
@@ -39,7 +38,7 @@ internal static IComWrapper TryGetWrapper(object managedObject)
                 var wrapper = Marshal.GetObjectForIUnknown(ptr);
 
                 // The aggregator (if there is one) implement IComWrapper!
```

Comment:
nit: Should you sweep comments? It seems silly to say 'implements IComWrapperFixed', when its the same interface (by GUID), but it may also be confusing otherwise? (or, more confusing, its already confusing, but necessarily so).

Reply:
Comments have now been updated